packages/core/src/routes/custom-phrase.ts

```diff
@@ -67,7 +67,7 @@ export default function customPhraseRoutes<T extends AuthedRouter>(
       params: object({ languageTag: languageTagGuard }),
       body: translationGuard,
       response: CustomPhrases.guard,
-      status: [200],
+      status: [200, 400],
```

Comment:
Should it be 500 instead of 400 if the translation script structure are conflicted?

Reply:
I think we should avoid using 500+ error codes since those could crash the app? Maybe 422 to indicate that the payload is not right.